### DISCUSSION TITLE
Documentation occludes all my previous code

### DISCUSSION THREAD

**Discussion by alper** [helix-editor/helix#9091] [General]
Is it supposed to be like this where the documentation occludes the previous 10+ lines of code I just wrote? I kinda need to be able to see those to figure out what I'm writing.

![CleanShot 2023-12-15 at 00 29 02@2x](https://github.com/helix-editor/helix/assets/72429/de5d362e-c85c-497b-8965-924e9b771115)

**Comment by kirawi** [comment]
You can disable `auto-signature-help` in your config

**Reply by gabydd** [reply]
You can also use ``C-c`` to dismiss the popup

**Reply by alper** [reply]
I think it's useful, just wish it wouldn't block the view of everything.

I see vscode starts the box at the cursor column. Or alternately would it be possible to render the box under the line instead of above?

**Reply by rojebd** [reply]
I disabled the option but, sometimes I  want to be able to see that many docs is there a way to have this toggleable?, I have it disabled by default in my config but toggle it for the current piece of code I'm hovering?

**Reply by kirawi** [reply]
Bind `signature_help`

**Reply by rojebd** [reply]
Thanks

**Reply by David-Else** [reply]
@alper 

> I think it's useful, just wish it wouldn't block the view of everything. 

Things should be much improved with https://github.com/helix-editor/helix/pull/6417
